Azure/sonic-utilities
# [201911] [Flex Counters] add CLI for PG drop packets counters (counterpoll, show/clear counters)

Should be merged after https://github.com/Azure/sonic-swss/pull/2263

Appropriate PR in master:https://github.com/Azure/sonic-utilities/pull/1355, https://github.com/Azure/sonic-utilities/pull/1461, https://github.com/Azure/sonic-utilities/pull/1583



#### What I did
Added new option for "counterpoll" utility
Added new CLI commands to view and clear PG dropped packet statistics.
Added the new CLI commands to the command reference guide.

#### How I did it
Need to merge PG drop functionality to 201911 

#### How to verify it
```
admin@arc-switch1041:~$ counterpoll pg-drop enable  --> to enable the new counter
admin@arc-switch1041:~$ counterpoll show  --> check new INGRESS_PG_STAT_DROP counter status
```

Check counters
```
admin@arc-switch1041:~$ redis-cli -n 2
127.0.0.1:6379[2]> HGETALL COUNTERS:oid:0x1a000000000062
1) "SAI_INGRESS_PRIORITY_GROUP_STAT_XOFF_ROOM_WATERMARK_BYTES"
2) "0"
3) "SAI_INGRESS_PRIORITY_GROUP_STAT_SHARED_WATERMARK_BYTES"
4) "0"
5) "SAI_INGRESS_PRIORITY_GROUP_STAT_DROPPED_PACKETS"
6) "0"
```

**show priority-group drop counters**
```
Ingress PG dropped packets:
     Port    PG0    PG1    PG2    PG3    PG4    PG5    PG6    PG7
---------  -----  -----  -----  -----  -----  -----  -----  -----
Ethernet0    800    801    802    803    804    805    806    807
Ethernet4    400    401    402    403    404    405    406    407
Ethernet8    100    101    102    103    104    105    106    107
...
```
**sonic-clear priority-group drop counters**
`Cleared PG drop counters`

#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
